api/authenticator.py

```diff
@@ -123,32 +123,44 @@ def authenticated_patron(self, _db, header):
                     return provider.authenticated_patron(_db, provider_token)
         return None
 
+    def _redirect_with_error(self, redirect_uri, problem_detail):
+        problem_detail_json = json.dumps(dict(uri=problem_detail.uri, title=str(problem_detail.title), detail=str(problem_detail.detail)))
```

Comment:
Why do this instead of calling the `json` function in core/util/problem_detail.py?